samples/scripts/syncAnnotationsScript.py

```diff
@@ -0,0 +1,261 @@
+import json
+import os
+
+import click
+import girder_client
+
+"""
+This script is used to sync a folder Hierarchy of Annotation files with a similar folder structure within Girder
+
+Arguments: 
+- folder: local Folder Location
+- girder_id: girderId for matching folder on the server
+
+Process:
+
+1.  The system will look through the current folder for CSV files.  It is specifically looking for files with
+The format of /folder/folder/{keyFolderName}/{label}/{keyFilename}Dataset.csv where the keyFolderName and keyFilename are specified below in the main settings
```

Comment:
```suggestion
The format of */{keyFolderName}/{label}/{keyFilename}Dataset.csv where the keyFolderName and keyFilename are specified below in the main settings
```

Reply:
This was leftover when they wanted to mirror the entire path.  Decided it was easier to provide the lower-level folder directory and use that.  Forgot to update that text.